content/sips/sip-325.md

```diff
@@ -0,0 +1,69 @@
+# SIP: 325
```

Comment:
please include the header in the below form
```
---
sip: 299
title: Minor upgrades to v2x to support V3 Migration
network: Ethereum & Optimism
status: Implemented
type: Governance
author: Daniel Beal (@dbeal-eth)
implementor: Daniel Beal (@dbeal-eth), MEB (@barrasso)
created: 2022-09-28
---
```

Reply:
this tool can also be used to setup a sip https://pr.synthetix.io/ in a seamless way, setup by the grants council